src/app/account/PageRegister.tsx

```diff
@@ -55,13 +56,13 @@ export const PageRegister = () => {
         description: title,
       });
 
-      if (errorKey === 'userexists') {
+      if (errorKey === ACCOUNT_ERROR_KEYS.loginAlreadyUsed) {
```

Comment:
please use uppercase and TS

Reply:
This instruction is based on the feedback of @ivan-dalmet  ([link](https://github.com/BearStudio/start-ui-web/pull/80#pullrequestreview-722086423)) , i see that this information could be important, if it is not important, I will be happy to apply this feedback.

Reply:
I agree with @Jessy-BAER , can you try to do it with TypeScript only. Challenge it please to try to avoid JavaScript.

Reply:
If i understand, you wanna me to specify the type of `error Key`, to let the TypeScript propose to the developer the possibilities of `errorKey`, and detect if ther's an error while typing the test of `errorKey`.

Reply:
@yoannfleurydev to check if I understood correctly, I have to add a type, and apply this type with axios to get a good typing with Typescript, to give the possibility of detecting errors at the `errorKey` at test level, and show error message if developer entered a wrong test.